Kickstarter-iOS/DataSources/ProjectPamphletContentDataSource.swift

```diff
@@ -49,28 +51,16 @@ internal final class ProjectPamphletContentDataSource: ValueCellDataSource {
       inSection: Section.subpages.rawValue
     )
 
-    self.setRewardTitleArea(project: project)
-
-    let rewardData = project.rewards
-      .filter { isMainReward(reward: $0, project: project) }
-      .sorted()
-      .map { (project, Either<Reward, Backing>.left($0)) }
-
-    if !rewardData.isEmpty {
-      if visible {
-      self.set(values: [project],
-               cellClass: RewardsTitleCell.self,
-               inSection: Section.rewardsTitle.rawValue)
-      }
-      self.set(values: availableRewards(for: project),
-               cellClass: RewardCell.self,
-               inSection: Section.availableRewards.rawValue)
-      self.set(values: unavailableRewards(for: project),
-               cellClass: RewardCell.self,
-               inSection: Section.unavailableRewards.rawValue)
+    if featureNativeCheckoutEnabled() {
+      self.setRewardTitleArea(project: project)
+      self.setRewards(project: project, visible)
     }
   }
 
+  private func featureNativeCheckoutEnabled() -> Bool {
+    return AppEnvironment.current.config?.features[Feature.checkout.rawValue] != .some(true)
```

Comment:
This logic always breaks my brain a bit 😅 . so if the flag is NOT `.some(true)`, it's returning `true`? Should it not be `== .some(true)`?

Reply:
Yep you're right, or rather the naming is confusing - this should either be `featureNativeCheckoutEnabled` -> `== .some(true)` OR `featureNativeCheckoutDisabled` -> `!= .some(true)` 😅 

Updating to `== .some(true)` which will change it to `if !featureNativeCheckoutEnabled()` at the call site.